docroot/themes/contrib/civictheme/includes/mobile_navigation.inc

```diff
@@ -37,20 +37,28 @@ function _civictheme_preprocess_block__civictheme_mobile_navigation(&$variables)
     ) {
       $menu_block = $entity->get($menu_field)->referencedEntities()[0];
       $menu_name = str_replace('menu_block:', '', $menu_block->get('settings')['id']);
-      $root = empty(str_replace($menu_name . ':', '', $menu_block->get('settings')['parent'])) ? '' : $menu_block->get('settings')['parent'];
+      $root = empty(str_replace($menu_name . ':', '', $menu_block->get('settings')['parent'] ?? '')) ? '' : $menu_block->get('settings')['parent'];
+
       $menu_parameters = new MenuTreeParameters();
-      $menu_parameters->setMaxDepth($menu_block->get('settings')['depth']);
+      // Only set max depth if the depth was provided (greater than 0).
```

Comment:
this is actually so weird - one part of Drupal thinks that `0` means `all levels`, but another part that `0` is actually depth of `0`. o_O